### ISSUE TITLE
[Vulkan] Sprites in Tomb Raider are corrupt

### ISSUE THREAD

**Issue by Papermanzero** [libretro/beetle-psx-libretro#320]
The sprites in Tom Raider (like stones or lianas) are completely corrupted. They are instantly moving and shaking.

**Comment by sepam21** [comment]
Turning off the PGXP vertex cache fixes this issue

**Comment by saftle** [comment]
This should be fixed with the last commit. Have you tested this again @Papermanzero?

**Comment by Papermanzero** [comment]
Not with vulkan. But the behavior is the same now in opengl.

**Comment by twinaphex** [comment]
I don't think this issue happens anymore. Please review it again @Papermanzero .

**Comment by Papermanzero** [comment]
The sprites are still shaking around
